src/cli.js

```diff
@@ -11,10 +11,52 @@ import fs from 'fs';
 import ciChoices from '../ci-choices.json';
 import containerizationChoices from '../containerization-choices.json';
 
+function initDocker(config) {
+  console.log('Creating Dockerfile and .dockerignore...');
+
+  const dockerFileContents = `# Use the official Node runtime as a parent image
```

Comment:
You should move this to a different file. Just create a new folder that can contain these files - Dockerfile, travis, etc etc